### DISCUSSION TITLE
Light Bulb Suggestion

### DISCUSSION THREAD

**Discussion by TaylordFed** [tuya/tuya-homebridge#336] [Ideas]
I have many lightbulbs added to tuya and added to the home app, but I do not have smart switches so whenever you turn the physical switch off to the smart bulb it remains "on" when it is really off. Maybe you could have it turn the switch to off whenever the device is offline. And if you try to turn it on when it is offline, the switch will just put itself back to the off position.

**Comment by LukaszKrawczakSP** [comment]
I have similar problem. When I turn switch on/off via physical device (remote pilot in my case) homebridge doesn't catch up new status and still shows old (wrong status). After homebridge reboot, homebridge fetches correct status.
